### PR TITLE
feat: first draft of the recipe

### PR DESCRIPTION
**Pull request by mostaphaRoudsari**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,29 @@
-# annual-radiation
+# Annual Radiation
+
 Annual radiation recipe for Pollination
+
+This recipe calculate the annual radiation for each timestep provided by the wea file.
+The outputs are stored under `results/direct` and `results/total`. Nigh-time hours are
+filtered before running the simulation. To match the results for eah hours see the list
```
Comment (chriswmackey) [inline README.md]:
```suggestion
filtered before running the simulation. To match the results for eah hours see the list
```
```suggestion
filtered before running the simulation. To match the results for each hours see the list
```

File: README.md [hunk 2]
```diff
@@ -1,2 +1,29 @@
...
+```
+Total radiation = direct sun radiation + indirect sky radiation
+```
+
+The number of bounces for direct radiation is set to 0 and cannot be modified by user.
+You can adjust the number of bounces for indirect sky radiation.
+
+# Limitations
+
+This recipe is limited to calculating up to `9999` number of suns at a time. This means
```
Comment (chriswmackey) [inline README.md:24]: Looking forward to removing this!!!

File: README.md (hunk 1 above)
Comment (mostaphaRoudsari) [reply README.md]: Thanks!

File: README.md (hunk 2 above)
Comment (mostaphaRoudsari) [reply README.md:24]: I tested it with the latest version and there was a bug! Greg has already fixed it. I'll test it when the new version is built tonight or tomorrow.

**Pull request merged**